src/Containers/CadView.jsx

```diff
@@ -559,6 +568,11 @@ export default function CadView({
         {alert}
       </>
       <SideDrawerWrapper/>
+      <input
```

Comment:
Per convo, please add/remove dynamically for only local files

Reply:
i have done.